.github/workflows/ci.yml

```diff
@@ -110,12 +110,12 @@ jobs:
               mkdir generators
               cmake --install build --prefix generators --component generators
               emcmake cmake -DSLANG_GENERATORS_PATH=generators/bin --preset emscripten -G "Ninja" -DSLANG_SLANG_LLVM_FLAVOR=DISABLE
-              cmake --build --preset emscripten --target slang
-              [ -f "build.em/Release/lib/libslang.a" ]
-              [ -f "build.em/Release/lib/libcompiler-core.a" ]
-              [ -f "build.em/Release/lib/libcore.a" ]
+              cmake --build --preset emscripten --config "$cmake_config" --target slang
+              [ -f "build.em/$cmake_config/lib/libslang.a" ]
+              [ -f "build.em/$cmake_config/lib/libcompiler-core.a" ]
+              [ -f "build.em/$cmake_config/lib/libcore.a" ]
           else
-            if [[ "${{ matrix.os }}" =~ "windows" && "${{ matrix.config }}" != "release" ]]; then
+            if [[ "${{ matrix.os }}" =~ "windows" && "${{ matrix.config }}" != "release" && "${{ matrix.config }}" != "releaseWithDebugInfo" ]]; then
```

Comment:
I think we can keep both cases "release" and "releaseWithDrbugInfo", here.
```
&& "${{ matrix.config }}" != "release" && "${{ matrix.config }}" != "releaseWithDebugInfo"
```

Reply:
sorry, what do you mean?

Reply:
Oops. Sorry I read the diff incorrectly.
Please ignore my comment above.